docs/UserGuide.md

```diff
@@ -296,31 +296,21 @@ Each candidate is uniquely identified by the same name, email and phone number.
 Manage a list of interviews for you to select the desired candidates, with the simple instructions below!
 
 #### <u>Add an interview:</u> `add_i`
+Use the following command to record the details of an interview session with the candidate(s) for a position!
 
 *Adds an interview to the list of interviews.*
 
 <u>Format:</u>
 
-    add_i position=<POSITION> index=<INDEX>... date=DATE time=TIME duration=DURATION [interviewed=STATUS]
+    add_i position=<POSITION> [c=<INDEX>]... date=DATE time=TIME duration=DURATION [interviewed=STATUS]
 
 <u>Example:</u>
 
     add_i position=Accountant c=1 2 date=18/10/2021 time=1400 duration=120 interviewed=pending
 
-* Adds an interview with the position of Accountant and the 1st and 2nd candidate in the list.
-* `POSITION` must be added to HR Manager and must have been applied by corresponding candidates before it can be used as a parameter.
-  * e.g., if the position, `Accountant` has not been added to HR Manager, the above command will result in an error : `Position Accountant not found in HR Manager`
-* `DATE` must be in numbers in DD/MM/YYYY form and can tolerate single digit for day and month, but year must be 4 digits.
-  * e.g., if the date, `2021/10/18` was used instead, HR Manager will show an error : `Date should be be valid and in DD/MM/YYYY format.`
-  * e.g., if the date, `18 Oct 21` was used instead, HR Manager will show an error : `Date should be be valid and in DD/MM/YYYY format.`
-* `TIME` must be in HHMM form, following 24-hour form, e.g., `1800` and `0600` for 6 P.M. and 6 A.M. respectively
-  * e.g., if the time, `6pm` was used instead, HR Manager will show an error : `Time should be be valid and in HHMM format..`
-* `DURATION` must be in numbers and is set to be in minutes
-  * e.g., if the duration, `twenty` was used instead, HR Manager will show an error : `Duration should be in numbers.`
-* `STATUS` must be either `pending` or `completed`
-  * e.g., if the status, `tbc` was used instead, HR Manager will show an error :`Interview Status can ony take the values:pending completed`
-    <br>
-    <br>
+* Adds a 120-minutes interview session on 18th October 2021 at 14:00 for the position of Accountant with the 1st and 2nd candidate from the list of candidates.
+
+Click [here](#Table of Inputs for Interview Management) to see the constraints and examples for the possible inputs.
```

Comment:
Does the hyperlink work locally in intellij? Cause in github, it doesn't seem to work

![image](https://user-images.githubusercontent.com/61644924/140009744-7c1f685b-d8d1-45bf-9775-5fce91ae82bd.png)

Reply:
Similar to the caution and info, none of the hyperlinks work locally, so I thought of checking after pushing it and before merging

Reply:
Just made it a draft, I think I'll reopen it when I solve the issue